src/args.jl

```diff
@@ -2161,7 +2161,7 @@ label_to_string(label, series_plotindex) = string(label)  # Fallback to string p
 function _update_series_attributes!(plotattributes::AKW, plt::Plot, sp::Subplot)
     pkg = plt.backend
     globalIndex = plotattributes[:series_plotindex]
-    plotIndex = _series_index(plotattributes, sp)
```

Comment:
Something is not cohesive here: You delete `sp` here, but `plotattributes` in the definition

Reply:
I think perhaps you misread? I split updating from getting the index. Currently not sure where to put the update call.

Reply:
now I see, thanks for clarifying